components/applicationsets/ztp-policies-appset.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: policies-sub
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: policies
+  namespace: openshift-gitops
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: policies-sub
+  project: policy-app-project
+  source:
+    path: clusters/ztp-overlays/policies/*
```

Comment:
Could do with documentation in ztp overlays. To describe at least a little what this intended to be in that dir. Also this is an application and not an application set so the name is a little misleading.

Reply:
I think I will add this with the policies work. So this could be added in another PR